man/systemd-keyutil.xml

```diff
@@ -64,6 +64,16 @@
 
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><command>pkcs1-to-pkcs7</command> <replaceable>PATH</replaceable></term>
+
+        <listitem><para>This commands converts an input signature in PKCS#1 format (RSA) to an
```

Comment:
I don't think we care about the key format? I'd drop the mention that this is RSA here.

Reply:
PKCS#1 is RSA https://en.wikipedia.org/wiki/PKCS_1